sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1160,7 +1170,23 @@ The `--all` flag can only be used with the `sem get jobs` command in order to
 display the most recent jobs of the current organization, both running and
 finished.
 
-### The EDITOR environment variable
+
+### Defining an editor
+
+The `sem` utility chooses which editor to use by following the next process:
+
+* From the value of the `editor` property in `~/.sem.yaml`, if it exists
+* From the value of the `EDITOR` environment variable, if it is set
+* If none of the above exists, `sem` will try to use the `vim` editor
+
+#### The editor property in .sem.yaml
```

Comment:
Lets advise the usage of `sem config set editor nano` instead of directly pointing the users to the Yaml config file.

Reply:
Did not know about this command - thanks!